bin/ch/WScriptJsrt.cpp

```diff
@@ -515,13 +515,24 @@ bool WScriptJsrt::PrintException(LPCWSTR fileName, JsErrorCode jsErrorCode)
             {
                 JsPropertyIdRef stackPropertyId = JS_INVALID_REFERENCE;
                 JsValueRef stackProperty = JS_INVALID_REFERENCE;
+                JsValueRef errorString = JS_INVALID_REFERENCE;
+                JsValueType stackPropertyType;
                 LPCWSTR errorStack = nullptr;
                 size_t errorStackLength = 0;
 
                 IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"stack", &stackPropertyId), false);
                 IfJsrtErrorFail(ChakraRTInterface::JsGetProperty(exception, stackPropertyId, &stackProperty), false);
-                IfJsrtErrorFail(ChakraRTInterface::JsStringToPointer(stackProperty, &errorStack, &errorStackLength), false);
+                IfJsrtErrorFail(ChakraRTInterface::JsGetValueType(stackProperty, &stackPropertyType), false);
 
+                if (stackPropertyType == JsUndefined)
+                {
+                    IfJsrtErrorFail(ChakraRTInterface::JsConvertValueToString(exception, &errorString), false);
```

Comment:
I guess the behavior should be; if `message` is defined, print `name` : `message` otherwise print exception->toString anyways ?